testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -157,14 +157,15 @@ def test_toggle_atom_nomatch_mismatch_atoms(self, universe, reference):
         with pytest.raises(SelectionError):
             align.alignto(u, ref, select='all', match_atoms=False)
 
-    @pytest.mark.parametrize('subselection, expectation', [
-        ('resname ALA and name CA', does_not_raise()),
-        (mda.Universe(PSF, DCD).select_atoms('resname ALA and name CA'), does_not_raise()),
-        (1234, pytest.raises(TypeError)),
-    ])
-    def test_subselection_alignto(self, universe, reference, subselection, expectation):
-
-        with expectation:
+    @pytest.mark.parametrize('subselection',
+        ['resname ALA and name CA', mda.Universe(PSF, DCD).select_atoms('resname ALA and name CA')])
+    def test_subselection_alignto_works(self, universe, reference, subselection):
+        rmsd = align.alignto(universe, reference, subselection=subselection)
+        assert_almost_equal(rmsd[1], 0.0, decimal=9)
+
+    @pytest.mark.parametrize('subselection', [1234])
+    def test_subselection_alignto_fails(self, universe, reference, subselection):
+        with pytest.raises(TypeError):
```

Comment:
What's the reason for this change? Looks unrelated to the reminder of the PR to me. What am I missing?

Reply:
Ideally existing tests shouldn't need to be changed. If it's something that needs fixing, a separate PR would probably be best

Reply:
False alarm, reverted.